src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private Color color;
+    private int side;
+    private int height;
+
+    public IsoscelesTrapezoid(Color color, int side, int height) {
+        this.color = color;
+        this.side = side;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return ((side + side) / 2) * height;
```

Comment:
integer division in floating-point context .
Please pay attention to [this](https://www.javatpoint.com/java-int-to-double) article.